gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Automating and Scaling Vex Generation"
+date: 2023-03-06T09:00:00Z
+draft: true
```

Comment:
I picked a time, but we can adjust it if necessary. 
"draft" must also be changed to "false"